### ISSUE TITLE
Very few contaminant detected regarding to the total number of ASVs

### ISSUE THREAD

**Issue by Mcouedel** [benjjneb/decontam#121]
Hi !
I had the idea to filter my ASVs from a threshold number of reads determined from my controls (>10 reads). I have 24 controls for 148 samples.  However, I was advised to use the decontam package to filter the data. I ran it on both my COI and 18S datasets, but I I have doubts about the number of contaminants found:

I show you my results:
For COI :
- With a detection threshold of 0.1 I get: 339,895 ASVs and 1 contaminant 
 - for a threshold of 0.2: 339,874 ASVs and 22 contaminants
 - for a threshold of 0.5 : 339,847 ASVs and 49 contaminants

For the 18S :
- threshold 0.1: 338,766 ASVs and 89 contaminants
- threshold 0.5: 338,227 ASVs and 628s contaminants
I get the same results in terms of number of contaminants if I filter out ASVs <2 samples
- threshold 0.1: 60 458 ASV and 89 contaminants
- threshold 0.5: 59,919 ASV and 628 contaminants

My dataset includes a majority of ASVs that are present in only 1 sample, can this play a role in the detection of contaminants by decontam?

For informations:
if I filter out ASVs <10 reads: 118,077 ASVs
if I filter out ASVs <2 samples + <10 reads: 16,389 ASV (my original filtration)

Best regards

**Comment by benjjneb** [comment]
> My dataset includes a majority of ASVs that are present in only 1 sample, can this play a role in the detection of contaminants by decontam?

Yes this will affect the results of decontam, as no single-sample ASVs will be classified as contaminants. decontam classifies features as contaminants if there is sufficient evidence to do so (modulable by the threshold). There is never sufficient evidence to do so when features appear in only one sample.

If you are working in a reasonably high biomass environment, then there may be few contaminant features in your data. That said, an additional filtration step removing ASVs with very little weight is likely a good idea prior to performing most downstream analyses. What information is an ASV appearing in just 1 sample going to provide to your analysis?
